### PR TITLE
Chore refractor

### PR DESCRIPTION
**Pull request by joaosilvalopes**

### CODE REVIEW

File: apply/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    while(maxPages--) {
+        await fillFields(page, formData).catch(noop);
+
+        await clickNextButton(page).catch(noop);
+
+        await waitForNoError(page).catch(noop);
+    }
+
+    try {
+        //await submit(page);
```
Comment (pmdacosta) [inline apply/index.js:35]: This should be set in the dotenv with an if condition to check if submit is on

File: fetch/fetchJobLinksUser.js [hunk 2]
```diff
@@ -11,26 +12,26 @@ async function fetchJobLinksUser({ page, location, keywords, remote, easyApply,
...
   await page.goto(url, { waitUntil: "load" });
 
-  const numJobsHandle = await page.waitForSelector('small.jobs-search-results-list__text', { timeout: 5000 });
+  const numJobsHandle = await page.waitForSelector(selectors.searchResultListText, { timeout: 5000 });
   const numJobs = await numJobsHandle.evaluate((el) => parseInt(el.innerText.replace(',', '')));
-  const jobTitleRexExp = new RegExp(jobTitle, 'i');
+  const jobTitleRegExp = new RegExp(jobTitle, 'i');
 
-  while (i < 64) {
+  while (i < 1) {
```
Comment (pmdacosta) [inline fetch/fetchJobLinksUser.js:19]: Another variable that could be set in dotenv

**Review by pmdacosta — APPROVED** [review]
Looks good

File: apply/index.js (hunk 1 above)
Comment (joaosilvalopes) [reply apply/index.js:35]: Yup I will do this in a separate pr with an argv flag

**Pull request merged**